src/richie/apps/courses/templates/courses/cms/program_detail.html

```diff
@@ -1,6 +1,11 @@
 {% extends "richie/fullwidth.html" %}
 {% load cms_tags extra_tags i18n thumbnail %}
 
+{% block head_title %}
+    {% page_attribute "page_title" as category_title %}
+    {% blocktrans with category_title=category_title %}{{ category_title }} - Program{% endblocktrans %}
```

Comment:
Copy pasta / failed ^^
```suggestion
    {% page_attribute "page_title" as program_title %}
    {% blocktrans with program_title=program_title %}{{ program_title }} - Program{% endblocktrans %}
```

Reply:
whoopsy